deploy/example_catalog/cr-app-flask-inf-engine.json

```diff
@@ -0,0 +1,130 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
```

Comment:
Let's not call it flask, should name ML deployment or something, also the file name should not say flask. Flask is an implementation detail